addons/web/static/src/views/kanban/kanban_controller.scss

```
@@ -103,8 +106,17 @@
             font-size: var(--Card-Footer-font-size, 1rem);
         }
 
+        aside {
+            flex: 1 0 var(--KanbanAside-width);
```

Why an `<aside/>` should be  allowed to grow outside the width defined by `--KanbanAside-width` 🤔 ?
I think that `<main/>` should be `flex-grow: 1` and this line should probably be something like 
```suggestion
            flex-basis: var(--KanbanAside-width);
```
could you check?